python/orca/src/bigdl/orca/ray/ray_on_spark_context.py

```diff
@@ -329,7 +329,7 @@ def _start_ray_services(iter):
 class RayOnSparkContext(object):
     _active_ray_context = None
 
-    def __init__(self, sc, redis_port=None, password="123456", object_store_memory=None,
+    def __init__(self, sc, redis_port=None, password=None, object_store_memory=None,
```

Comment:
In line 349, change the default value to None. 😄

Reply:
Good catch! updated! thx!

Reply:
None and "None" are different